code/style.css

```diff
@@ -1,7 +1,205 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+/*  Remove Existing Styling */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+    text-decoration: none;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.header-image {
+    height: 50vh;
+    width: 100vw;
+    background-image: url(./images/schnauzer-pexels-sebastian-coman.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
```

Comment:
The Hero image is perfectly fine for iphone and ipad device but for Desktop you can only see a half dog. Perhaps a solution is to add some media querys for the desktop version.